parse.go

```diff
@@ -0,0 +1,53 @@
+package vagrantutil
+
+import (
+	"bufio"
+	"strings"
+)
+
+// parseData parses the given vagrant type field from the machine readable
+// output (records).
+func (v *Vagrant) parseData(records [][]string, typeName string) (string, error) {
```

Comment:
I'm printing all the errors in debug mode [here](https://github.com/koding/vagrantutil/pull/5/files#diff-090598d65549267a01d4ed4132809255R29), which needs access to `v`'s logger.